gatsby-config.js

```diff
@@ -1,3 +1,10 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+require("dotenv").config({
+  path: `.env.${process.env.NODE_ENV}`,
+})
+
+const facebookPageID = 431794124327063 // ReactJS PH Facebook Page ID
```

Comment:
**`NITPICK`** Can you use constant case?

Reply:
Sure! 😄